aria-practices.html

```diff
@@ -2150,6 +2150,37 @@ <h4>WAI-ARIA Roles, States, and Properties</h4>
       </section>
     </section>
 
+    <section class="widget" id="meter">
+      <h3>Meter</h3>
+      <p>
+        An <a class="role-reference" href="#meter">meter</a> is an element that allows authors to define a scalar value within known minimum and maximum limits.
+      </p>
+      <p class="note">
+        The <code>meter</code> widget should not be used to indicate progress, such as loading or percent completion of a task. For that, use the <a href="#progressbar" class="role-reference">progressbar</a> role instead.
+      </p>
+
+      <!-- <section class="notoc">
+        <h4>Example</h4>
+        <p>TODO: add example link</p>
+      </section> -->
+
+      <section class="notoc">
+        <h4>Keyboard Interaction</h4>
+        <p>A meter does not require any keyboard interaction.</p>
+      </section>
+
+      <section class="notoc">
+        <h4>WAI-ARIA Roles, States, and Properties</h4>
+        <ul>
+          <li>The widget has a role of <a class="role-reference" href="#meter">meter</a>.</li>
+          <li>The meter element has <a class="property-reference" href="#aria-valuenow">aria-valuenow</a> set to a decimal value between <code>aria-valuemin</code> and <code>aria-valuemax</code> representing the current value of the meter.</li>
```

Comment:
Not sure we should say "the meter element", just in case authors think we mean "the html meter element"?
In other patterns, we have said either:
- "the element with role meter", or
- "the meter"

(5 instances)

Reply:
I'm happy to change this, though if it's a general guidance it might be good to revisit some of the other pattern wording, e.g. `radiobutton`, `slider`, `spinbutton`, and `menu` (there are probably more, I didn't look through comprehensively. Those are the ones I looked at when writing meter)